core/gatewayclient/example/go.mod

```diff
@@ -1,21 +1,22 @@
 module mykgwapp
 
-go 1.21.0
+go 1.22
 
-require github.com/kwilteam/kwil-db/core v0.1.1-0.20240514193332-c61ba10f8a1c
+require github.com/kwilteam/kwil-db/core v0.3.0-beta.0.20240917145512-e6922733fc1a
```

Comment:
I see core/client uses a different version, should we keep them same?

Reply:
Yup, totally.  updated.